staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -44,6 +45,15 @@ const (
 // ServerRunOptions contains the options while running a generic api server.
 type ServerRunOptions struct {
 	AdvertiseAddress net.IP
+	// PeerCAFile is the ca bundle used by this kube-apiserver to verify peer apiservers'
+	// serving certs when routing a request to the peer in the case the request can not be served
+	// locally due to version skew.
+	PeerCAFile string
```

Comment:
validate this is not set if the UVIP feature is not enabled

Reply:
once the UVIP feature gate graduates to beta and enables by default, what will control enablement of the feature? populating this CA field?

Reply:
yes, since the other flags for --peer-advertise-ip / --peer-bind-port will be assigned some defaults.

Reply:
huh I guess validating that --peer-ca-file be set when the UVIP feature flag is enabled is not required? It breaks tests.